extensions/sdk/src/Rally.ts

```diff
@@ -137,6 +129,21 @@ export class Rally {
     ["source", "medium", "campaign", "term", "content"].forEach((key) => {
       attribution[key] = url.searchParams.get(`utm_${key}`);
     });
+
+    return attribution;
+  }
+
+  /**
+   * Attempt to fetch the attribution codes from the store page URL for this extension.
+   */
+  private async storeAttributionCodes() {
+    let attribution = await this.getAttributionCodes();
```

Comment:
This optimization can likely be moved to getAttributionFromStore method and we can just call the store function here?